cmd/node/main.go

```diff
@@ -766,11 +774,11 @@ func startStatisticsMonitor(file *os.File, config config.ResourceStatsConfig, lo
 	return nil
 }
 
-func createApiResolver() (facade.ApiResolver, error) {
-	//TODO replace this with a vm factory
-	vm := &mock.VMExecutionHandlerStub{}
+func createApiResolver(vmAccountsDB vmcommon.BlockchainHook) (facade.ApiResolver, error) {
+	cryptoHook := &hooks.VMCryptoHook{}
```

Comment:
leave the TODO as the factory is a must :P

Reply:
Re-added TODO